gossipd/gossipd.c

```diff
@@ -747,6 +747,42 @@ static void gossip_refresh_network(struct daemon *daemon)
 	route_prune(daemon->rstate);
 }
 
+static void tell_master_local_cupdates(struct daemon *daemon)
+{
+	struct chan_map_iter i;
+	struct chan *c;
+	struct node *me;
+
+	me = get_node(daemon->rstate, &daemon->id);
+	if (!me)
+		return;
```

Comment:
Do you think this warrants a log_unusual?

Reply:
No, it happens with no public channels...